scripts/scripts.js

```diff
@@ -132,15 +133,228 @@ export function decorateMain(main) {
   decorateQuickLinks(main);
 }
 
+function initWebSDK(path) {
+  return new Promise((resolve) => {
+    import(path)
+      .then(() => {
+      })
+      .then(() => {
+        resolve();
+      })
+      .catch(() => {
+      });
+  });
+}
+
+function onDecoratedElement(fn) {
+  // Apply propositions to all already decorated blocks/sections
+  const elements = document.querySelectorAll('[data-block-status="loaded"],[data-section-status="loaded"]');
```

Comment:
Why having sections and blocks if they overlap?

Reply:
Not all sections have blocks, and you want to be able to catch any of those as they are shown in the DOM to be able to patch them immediately and avoid the flicker